src/colors.tsx

```diff
@@ -0,0 +1,21 @@
+//Colors for Redesign
+
+const colors = {
```

Comment:
I keep seeing this file in the PRs. I think you are pointing to the wrong branch maybe

Reply:
@nicolam1 that's the new color scheme for the redesign I made initially with the new colors from FIgma. It's referenced in the `src/theme.tsx` file just to make the code a little more readable. I can get rid of it and just put the colors directly in the theme file if you want? The `colors.tsx` file is already in master though, this branch is able to merge with master.

Reply:
@willmccormick24 If this was in master then we wouldn't see this change in the DIFF, maybe rebase the branch on master and push again